multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -402,6 +423,12 @@ func (r *ServiceExportReconciler) updateSvcExportStatus(ctx context.Context, req
 	case serviceNotFound:
 		newCondition.Reason = getStringPointer("service_not_found")
 		newCondition.Message = getStringPointer("the Service does not exist")
```

Comment:
And `Message` should be "Service does not exist"

Reply:
Updated